### ISSUE TITLE
Timeout for WinAppDriver in Java

### ISSUE THREAD

**Issue by leticiafer** [microsoft/WinAppDriver#1418]
Is it possible to do a timeout for WinAppDriver with this instruction in Java? `WindowsDriver<WindowsElement> session = new WindowsDriver<WindowsElement>(new Uri("http://127.0.0.1:4723"), appCapabilities, TimeSpan.FromMinutes(10));`

**Comment by anunay1** [comment]
no. This instruction is for the commands, what is the timeout that you are looking for.

**Comment by leticiafer** [comment]
I need timeout with WinAppDriver, that is to say, I need a wait for WinAppDriver while app is starting. I know thread.sleep but I need time in part of server (WinAppDriver) not in App.

**Comment by trashbat** [comment]
You can wait up to 50 seconds for the app to start with the latest version of WinAppDriver (1.2.1). Something like this in Java:

```java
appCapabilities.setCapability("ms:waitForAppLaunch", "50");
```
